premailer/premailer.py

```diff
@@ -540,6 +544,8 @@ def _load_external_url(self, url):
     def _load_external(self, url):
         """loads an external stylesheet from a remote url or local path
         """
+        if self.skip_load_external:
```

Comment:
Since it's not many places where `self._load_external()` is called, I think you can be confident it won't get here if `skip_load_external` *is* `False`.